voting/orbs/Elections/election_results.go

```diff
@@ -17,8 +17,8 @@ import (
 /*****
  * Election results
  */
-func getElectionPeriod() uint64 {
```

Comment:
all block based functions moved to block-based as a sign of "ready for deprecated".

Reply:
with this same name?

Reply:
these functions are mostly exported and although we don't have to i would rather not change their names just in case anyone already calls them in some guardian script.